docs/source/models/other_models/facility_choice/index.rst

```diff
@@ -11,93 +11,370 @@ Delivery Facility Choice Model
+<https://en.wikipedia.org/wiki/Polychoric_correlation>`_ between ordinal
+variables, and it relies on having a known ordering of each variable's
+values. We will follow the convention of ordering the categories of all
+categorical variables from "highest risk" to "lowest risk" (GBD often
+follows this convention for risk factors), so that larger propensities
+are generally "better" for the simulant.
+
+We use an ordering of the LBWSG categories that we hypothesize will make
+them have large polychoric correlation with the ANC and IFD
+propensities. Our chosen ordering also facilitates convergence of the
+causal model optimization, whose objective function involves the
+conditional probability of preterm status given facility choice.
+**Specifically, we order the LBWSG categories first by preterm status
+(preterm < full-term), then from highest average RR to lowest average RR
+in the early neonatal age group (averaged across all draws), separately
+for each sex.**
+
+.. important::
+
+  * All preterm categories (< 37 weeks) are ordered **before** all
+    full-term categories (37+ weeks)
+  * The ordering is **sex-specific** (the ordering is different for
+    males and females)
+  * Within each term status (preterm or full-term), LBWSG categories are
+    ordered in **decreasing** order by (sex-specific) average relative
+    risk across draws
+  * The ordering is based on the RRs for the **early neonatal** age
+    group since we're interested in the risk right after birth
+
+  This ordering must be used when initializing the LBWSG category from
+  its (correlated) propensity :math:`u_\text{cat}`, following the
+  strategy described on the :ref:`LBWSG risk exposure page
+  <2019_risk_exposure_lbwsg>`.
+
+**We will also order the ANC and IFD propensities from highest to lowest
+risk: "no ANC" < "some ANC"; and "home birth" < "in-facility birth".**
+These orderings must be used when initializing simulants' ANC status and
+IFD status from the corresponding (correlated) propensities
+:math:`u_\text{ANC}` and :math:`u_\text{IFD}`. See the :ref:`Antenatal
+care attendance module <2024_vivarium_mncnh_portfolio_anc_module>` for
+more details on assigning ANC status; see the `Causal conditional
+probabilities for in-facility delivery`_ section below for an explicit
+description of how to assign IFD status.
+
+To be more explicit about how the ordered categories and propensities
+work in code: If the categories are ordered from highest risk to lowest
+risk as :math:`c_1, \dotsc, c_n`, divide the unit interval :math:`[0,1]`
+into :math:`n` subintervals :math:`I_1, \dotsc, I_n` ordered from left
+to right, such that the length of :math:`I_j` is :math:`\Pr(c_j)`. Then
+a uniform propensity :math:`u \in [0,1]` corresponds to category
+:math:`c_j` precisely when :math:`u \in I_j`. This correspondence
+specifies how each ordinal variable should be initialized from its
+corresponding propensity. [[A picture would probably help, should we add
+one here?]]
```

Comment:
If helpful as visual inspiration, I think this is similar to choosing cause of death from multiple options with a propensity: https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_features/cause_of_death_selection/index.html#proposed-behavior

Reply:
Great, thanks Zeb! Yeah, something like those diagrams is what I had in mind. Perhaps I'll make a Jira ticket for adding a similar diagram here.